src/randoop/JunitFileWriter.java

```diff
@@ -36,10 +35,12 @@
 
   private Map<String, List<List<ExecutableSequence>>> createdSequencesAndClasses = new LinkedHashMap<String, List<List<ExecutableSequence>>>();
 
+  private Map<String, Integer> methodCountMap = new LinkedHashMap<String,Integer>();
```

Comment:
Please add a Javadoc comment for every method and field, to explain its meaning, purpose, or use.
(I realize this is not currently followed, but we can do better in code we are adding/changing.